### DISCUSSION TITLE
Stagger more "energizing" virtual coffee monthly events apart

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1186] [New Ideas]
I think it would be cool to stagger more chill VC monthly challenges with more energizing ones that build/lead up to some event at the end of the month.

Like every 3 months we have an energizing monthly challenge (like the lightning talks or hackathon) then we have two chill monthly challenges (like kindness and mid-year check in)

I don't know how much to stagger. Depends on how much bandwidth and interest everyone has. Could be every 2, 3, 4 months. But staggering is the idea

**Comment by BekahHW** [comment]
Some of this definitely depends on the community at the time. I would say our most well-participated challenges are:

- hacktoberfest
- NanoWriMo (blogging challenge in Nov)
- Lightning talks (this was the first year run as a challenge and not an event)
- Demo Days (build in public for a month and end with a demo) 

The challenge is that events like that take a lot of energy and coordination, so we'd definitely need to identify volunteers who could put in some extra time. 

What other challenges would you like to see? We have talked about a hackathon in the past, which could be really fun if someone would run it. 

I'll also tag in our monthly challenge leaders since they've put in a lot of work on evaluating the challenges, and tag in Meg as events maintainer. 

@meg-gutshall 
@adiati98 
@dominicduffin1
